.circleci/config.yml

```diff
@@ -1,43 +0,0 @@
-version: 2
```

Comment:
_technically_ you should have the file at this path as of 2.0...

But it has no discernible effect on anything